serverless.yml

```diff
@@ -149,6 +152,23 @@ resources:
         QueueName: ${self:custom.sqs}-dead-letter-queue
         MessageRetentionPeriod: 1209600
 
+    HitRate:
```

Comment:
This hitrate table collects rows for each item processed, so we can use timestamps to query a sliding window of hits to calculate current rate versus limit. The TimeToLive feature in DynamoDB will continually clean up old rows over time.